console-subscriber/src/builder.rs

```diff
@@ -149,6 +158,283 @@ impl Builder {
 
         self
     }
+
+    /// Initializes the console [tracing `Subscriber`][sub] and starts the console
+    /// subscriber [`Server`] on its own background thread.
+    ///
+    /// This function represents the easiest way to get started using
+    /// tokio-console.
+    ///
+    /// In addition to the [`TasksLayer`], which collects instrumentation data
+    /// consumed by the console, the default [`Subscriber`][sub] initialized by this
+    /// function also includes a [`tracing_subscriber::fmt`] layer, which logs
+    /// tracing spans and events to stdout. Which spans and events are logged will
+    /// be determined by the `RUST_LOG` environment variable.
+    ///
+    /// **Note**: this function sets the [default `tracing` subscriber][default]
+    /// for your application. If you need to add additional layers to a subscriber,
+    /// see [`spawn`].
+    ///
```

Comment:
nit: maybe worth adding:
```suggestion
    ///
    /// ## Panics
    ///
    /// * If the subscriber's background thread could not be spawned.
    /// * If the [default `tracing` subscriber][default] has already been set.
    ///
```